R/upsample.R

```diff
@@ -28,10 +28,7 @@
 #' @param target An integer that will be used to subsample. This
 #'  should not be set by the user and will be populated by `prep`.
 #' @param seed An integer that will be used as the seed when upsampling.
-#' @return An updated version of `recipe` with the new step
-#'  added to the sequence of existing steps (if any). For the
-#'  `tidy` method, a tibble with columns `terms` which is
-#'  the variable used to sample.
```

Comment:
this is the other place where the sentence on the `tidy()` method is removed

Reply:
Yes, since the "real" documentation now lives in themis 👍